packages/api-bzz-react-native/src/index.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-env browser */
+import * as stream from 'stream'
```

Comment:
Do the `stream` APIs work as expected with React-Native please? We don't have a test suite for RN so I'd like to make sure this doesn't break existing functionalities.

Reply:
Good question, I have no experience with RN, sooooo I will leave up to @vujevits to let me know. Although I believe this is only there for Typing support, maybe we could use `readable-stream` instead to make sure that it is truly compatible.